src/Expectation.php

```diff
@@ -66,6 +66,8 @@ public function and(mixed $value): Expectation
 
     /**
      * Creates a new expectation with the decoded JSON value.
+     *
+     * @return self<mixed>
```

Comment:
The mixed is always an array?

Reply:
Well, technically you could be json_decoding 'true', or 'false' or another string value, in which case it wouldn't return an array. Thoughts?